server.js

```diff
@@ -1,20 +1,18 @@
-'use strict';
+"use strict";
 
-const express     = require('express');
-const bodyParser  = require('body-parser');
-const fccTesting  = require('./freeCodeCamp/fcctesting.js');
+const express = require("express");
+const fccTesting = require("./freeCodeCamp/fcctesting.js");
 
 const app = express();
 
 fccTesting(app); //For FCC testing purposes
-app.use('/public', express.static(process.cwd() + '/public'));
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: true }));
+app.use("/public", express.static(process.cwd() + "/public"));
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
 
-app.route('/')
-  .get((req, res) => {
-    res.sendFile(process.cwd() + '/views/index.html');
-  });
+app.route("/").get((req, res) => {
+  res.render(process.cwd() + "/views/index");
```

Comment:
```suggestion
  res.sendFile(process.cwd() + "/views/index.html");
```
Changing this from `sendFile` to `render` is part of the challenge, so we should leave it for the user to do.

Reply:
related conversation on the approach for the template. 
https://github.com/freeCodeCamp/freeCodeCamp/pull/38572#issuecomment-616644655